docs/modules/release-notes/pages/5-4-1.adoc

```diff
@@ -0,0 +1,16 @@
+= 5.4.1 Release Notes
+
+== Enhancements
+
+* Introduced a new YAML configuration option for User Code Namespaces that enables multiple name spaces to be defined within a `name-spaces` collection [#2485]. For more info, see xref:clusters:ucn-data-structures.adoc[].
```

Comment:
James has updated examples in the following files:

- ucn-member-side.adoc
- ucn-non-associated.adoc

Reply:
Thanks, I added link to the member-side page as looked most appropriate.